src/h_assets/assets.py

```diff
@@ -72,65 +28,62 @@ def __init__(
         """
         Construct an Environment from the given configuration files.
 
-        :param assets_base_url: The URL at which assets will be served,
+        :param assets_base_url: URL at which assets will be served,
             excluding the trailing slash.
-        :param bundle_config_path: Asset bundles config file.
-        :param manifest_path: JSON file mapping file paths in the bundle config
-            file to cache-busted URLs.
-        :param auto_reload: If True the config and manifest files are
-            automatically reloaded if they change.
+        :param bundle_config_path: Asset bundles config file path.
+        :param manifest_path: JSON file mapping bundle names to files and
+            cache-busted URLs for them
+        :param auto_reload: Automatically reload the config and manifest files
+         if they change.
         """
 
-        # Parse an asset manifest. This doesn't actually validate the
-        # structure, it just documents the expected format.
-        def load_manifest(file_):
-            return json.load(file_)
-
         self.assets_base_url = assets_base_url
-        self.manifest = _CachedFile(
-            manifest_path, load_manifest, auto_reload=auto_reload
-        )
-        self.bundles = _CachedFile(
-            bundle_config_path, _load_bundles, auto_reload=auto_reload
-        )
+
+        self.manifest_file = CachedJSONFile(manifest_path, auto_reload=auto_reload)
+        self.bundle_file = CachedINIFile(bundle_config_path, auto_reload=auto_reload)
 
     def files(self, bundle):
         """Return the file paths for all files in a bundle."""
 
-        bundles = self.bundles.load()
-        return bundles[bundle]
+        return aslist(self.bundle_file.load()["bundles"][bundle])
```

Comment:
This leads to a little more work each time `urls` is called, but I'd guess the impact will be minimal.